### PR TITLE
Background Process: Create perflab_start_background_job function 

### LINKED ISSUE #480: Create Background Processing Infrastructure: Create perflab_start_background_job
Create a new `perflab_start_background_job()` function to start the background process for a specific job ID by making a HTTP POST request to the `admin-ajax.php` endpoint.

## Acceptance Criteria
- Function accepts the following parameters
    - `$job_id` - A integer that is the ID for a job returned by the `perflab_create_background_job` function
- The function will make a HTTP request to the `admin-ajax.php` endpoint with the `wp_remote_post` along with the following parameters
    - action: `wp_ajax_background_process_handle_request`
    - nonce: `wp_create_nonce( 'wp_ajax_background_process_handle_request' )`
    - timeout: `0.1`
    - data: `array( $job_id )`
    - blocking: `false`
    - sslverify: `apply_filters( 'https_local_ssl_verify', false )`
- The function will return the following depending on success or failure
    - On failure the function returns the `WP_Error` from `wp_remote_post`
    - On success, the function returns `true`

### PR DESCRIPTION
**Pull request by jjgrainger**
## Summary

* Adds the `perflab_start_background_job()` function

Fixes #480 

## Checklist

- [x] PR has either `[Focus]` or `Infrastructure` label.
- [x] PR has a `[Type]` label.
- [x] PR has a milestone or the `no milestone` label.

### CODE REVIEW

**Comment by ankitrox** [comment]
@jjgrainger I think this PR can be closed in favour of background process runner which already contains this function [here](https://github.com/WordPress/performance/pull/512/files#diff-97f552ccb63c9fdd70b8b372c33a768277cf6f894b01a51aea3f377790605683R68-R92)

**Comment by swissspidy** [comment]
Closing for now to clean up the repo since this plugin/module is not a priority right now.
